base/string.jl

```diff
@@ -968,8 +968,8 @@ byte_string_classify(s::ByteString) = byte_string_classify(s.data)
     # 1: valid ASCII
     # 2: valid UTF-8
 
-is_valid_ascii(s::Union(Array{UInt8,1},ByteString)) = byte_string_classify(s) == 1
-is_valid_utf8(s::Union(Array{UInt8,1},ByteString)) = byte_string_classify(s) != 0
+isvalid(::Type{ASCIIString}, s::Union(Array{UInt8,1},ByteString)) = byte_string_classify(s) == 1
```

Comment:
It grossly unfair to say that the Julia developers have not been responsive to you.   The core developers have donated a *lot* of time responding to your extremely numerous comments and requests, making suggestions, and helping your work on your PRs to get them into an acceptable form.  (Notice the key word, *donated*; you've been asking for a *lot* of time from people you aren't paying, and it's not particularly polite to complain that they don't donate more.)

On a technical note, if you add a new method to `convert`, it will be called by functions in Base as well as in your own code when that conversion is required.   You are extending the *existing* `convert` methods, and hence can extend the existing behavior; this is the whole point of *dynamic* multiple dispatch (as opposed to static dispatch as in e.g. C++).

Threatening to cease contributing if a particular PR is not merged quickly is an ultimatum, whether you call it one or not.